test/integration/fixtures/install-variants/bower-registry/bower.herokuapp.com/lookup/webcomponentsjs_7bba9

```diff
@@ -0,0 +1 @@
+{"expires":1489941177396,"value":{"type":"alias","url":"https://github.com/Polymer/webcomponentsjs.git"}}
```

Comment:
1489941177396 is this Sunday. Should we push this off to the impractically distant future?

2489941177396 is in 2048.

Also, how did you generate this?